### DISCUSSION TITLE
Didn't find LoadMore support in Listbox for async usecases in react aria components use cases

### DISCUSSION THREAD

**Discussion by RajaTheKing826** [adobe/react-spectrum#7055] [Q&A]
Didn't find LoadMore support in Listbox for async usecases in react aria components use cases, can you please tell me to how can i get it?
Documentation link -> https://react-spectrum.adobe.com/react-aria/ComboBox.html
code sandbox link -> https://codesandbox.io/p/sandbox/gallant-surf-j4wsct?file=%2Fsrc%2FRACComboBox.tsx%3A25%2C48

**Comment by snowystinger** [comment]
did you mean this? https://react-spectrum.adobe.com/react-aria/ComboBox.html#asynchronous-loading
I think you included the wrong sandbox link, it's the one that had to do with your other question about the menu width it looks like

**Reply by RajaTheKing826** [reply]
https://react-spectrum.adobe.com/react-aria/ComboBox.html#asynchronous-loading Here i didn't find _loadmore_ prop to **listbox** for infinite scroll async loading case.

**Reply by RajaTheKing826** [reply]
@snowystinger  did you get my doubt?

**Reply by snowystinger** [reply]
I apologize, but I am not following.

Are you trying to implement virtual scrolling with loadmore? We recently began supporting that, there are some examples in our code base and hopefully we'll have docs in the near future.
https://github.com/adobe/react-spectrum/blob/246225b096442a39f1f91b351f4ff88d9707ca2d/packages/%40react-aria/virtualizer/src/Virtualizer.tsx#L35

If you try to implement loadmore without virtualization, you may run the risk of slowing your app down with thousands of results. Though it should be doable by providing something like the Virtualizer.

Otherwise, can you put the loadmore prop where you expect it in the codesandbox? and how you're trying to use it? maybe that will clear things up.
